app/views/categories/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="d-flex flex-column justify-content-between align-items-between">
+  <h1 class="mb-3">Categories</h1>
+
+  <div>
+    <% @categories.each do |category| %>
+      <div class="card mb-1">
+        <div class="card-body">
+          <div class="row">
+            <div class="col">
+              <%= image_tag(category.icon, alt: "Icon pic", height: "40px") %>
+            </div>
+            <div class="col">
+              <%= link_to category.name, category_path(category) %>
+            </div>
+            <div class="col">
+              $<%= category.expenses.sum(:amount) %>
+            </div>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+  <%= link_to "Add new category", new_category_path, class: "btn btn-success mt-3" %>
```

Comment:
Please follow the design for Categories page. Kindly fix the button position on bottom of the page as it's requested by previous code reviewer.

Reply:
Thanks for the feedback.